examples/semi_supervised/plot_label_propagation_digits_active_learning.py

```diff
@@ -69,29 +72,35 @@
     pred_entropies = stats.distributions.entropy(
         lp_model.label_distributions_.T)
 
-    # select five digit examples that the classifier is most uncertain about
-    uncertainty_index = uncertainty_index = np.argsort(pred_entropies)[-5:]
+    # select up to five digit examples that the classifier is most uncertain about
+    uncertainty_index = np.argsort(pred_entropies)[::-1]
+    uncertainty_index = uncertainty_index[
+        np.in1d(uncertainty_index, unlabeled_indices)][:5]
 
     # keep track of indices that we get labels for
     delete_indices = np.array([])
-
-    f.text(.05, (1 - (i + 1) * .183),
-           "model %d\n\nfit with\n%d labels" % ((i + 1), i * 5 + 10), size=10)
+    
+    # if running for more than 5 iterations visualiaze the gain only on the first 5.
+    if i < 5:
+        f.text(.05, (1 - (i + 1) * .183),
+               "model %d\n\nfit with\n%d labels" % ((i + 1), i * 5 + 10), size=10)
     for index, image_index in enumerate(uncertainty_index):
         image = images[image_index]
 
-        sub = f.add_subplot(5, 5, index + 1 + (5 * i))
-        sub.imshow(image, cmap=plt.cm.gray_r)
-        sub.set_title('predict: %i\ntrue: %i' % (
-            lp_model.transduction_[image_index], y[image_index]), size=10)
-        sub.axis('off')
+        # if running for more than 5 iterations visualiaze the gain only on the first 5.
```

Comment:
Done.  I omitted the capitalization though since no other comments in the file are capitalized.